### PR TITLE
JIRA-5: Create entities and database setup

### PR DESCRIPTION
**Pull request by AndreySemeynikovIF**
[JIRA-5](https://ifortex-team-java1.atlassian.net/browse/EPS-5). Setup project, add configuration files. Added Student and Course model classes. Define db connection and setup migration script using flyway. Added stylsih changes into sql script and and moved it to the resources package.

### CODE REVIEW

**Comment by vladqf** [comment]
pom.xml file is missing

File: src/main/java/com/ifortex/internship/config/MyWebApplicationInitializer.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package com.ifortex.internship.config;
+
+import jakarta.servlet.ServletContext;
+import jakarta.servlet.ServletException;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.WebApplicationInitializer;
+
+@Configuration
+public class MyWebApplicationInitializer implements WebApplicationInitializer {
```
Comment (PavelShynkarou) [inline src/main/java/com/ifortex/internship/config/MyWebApplicationInitializer.java:9]: Looks like we figured out that this class unnecessarily.

File: src/main/java/com/ifortex/internship/model/Student.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package com.ifortex.internship.model;
+
+import lombok.*;
```
Comment (PavelShynkarou) [inline src/main/java/com/ifortex/internship/model/Student.java:3]: Could you please fix it?

**Review by PavelShynkarou — CHANGES_REQUESTED** [review]

File: src/main/java/com/ifortex/internship/config/MyWebApplicationInitializer.java (hunk 1 above)
Comment (AndreySemeynikovIF) [reply src/main/java/com/ifortex/internship/config/MyWebApplicationInitializer.java:9]: yep, deleted it.

File: src/main/java/com/ifortex/internship/model/Student.java (hunk 2 above)
Comment (AndreySemeynikovIF) [reply src/main/java/com/ifortex/internship/model/Student.java]: fixed. Also changed the setting in IDEA to prevent such case in the future.

**Review by PavelShynkarou — APPROVED** [review]

**Review by vladqf — APPROVED** [review]

**Pull request merged**